### PR TITLE
TG-49 getGameById and GET /games/:gameID ready to go

### PR DESCRIPTION
**Pull request by gideonw14**
Mostly just a copy pasta of TG-48 except with gameID instead.

### CODE REVIEW

File: db/teams.js [hunk 1]
```diff
@@ -100,9 +100,11 @@ function createTeam (teamName, email, password, isEligible) {
 }
 
 module.exports = {
-  createTeam,
-  getTeam,
-  getTeamByName,
-  editTeam,
-  getAllTeamNames
-}
+  createTeam: createTeam,
```
Comment (renodubois) [inline db/teams.js]: Same comment as the other PR, you should be able to change the format from `createTeam: createTeam` to just `createTeam`

**Review by renodubois — APPROVED** [review]
Looks about the same as the other PR, so going to approve this one as well.

**Comment by renodubois** [comment]
@michael-beaver If you can go over this whenever you get the chance, that would be great.

I think everything should be working fine now, but I want to have another set of eyes on it.

**Review by michael-beaver — APPROVED** [review]
I don't see any glaring errors, seems good

**Pull request merged**